pkg/cmd/admin/prune/prune.go

```diff
@@ -5,6 +5,7 @@ import (
 
 	"github.com/spf13/cobra"
 
+	"github.com/openshift/origin/pkg/cmd/admin/groups/sync/cli"
```

Comment:
alias this package to something more reasonable so that the call below says what it means.